README.md

```diff
@@ -430,6 +430,19 @@ server:
     # By default requests are accepted from all the IPs.
     allowed_networks: ["office", "reporting-apps", "1.2.3.4"]
 
+    # ReadTimeout is the maximum duration for reading the entire
+    # request, including the body.
```

Comment:
Could you specify on which side this timeout is applied - on the chproxy client or on the clickhouse server? Probably, it would be better adding the corresponding prefix to these timeouts? For instance, `client_read_timeout`

Reply:
That's the good point! I was sure being a part of `server.http` and/or `server.https` section was clear enough. But adding a prefix to the properties names could be a bit verbose. I suggest to specify description instead. Do you agree?

Reply:
Yes